src/Params/PADnoteParameters.h

```diff
@@ -176,6 +176,27 @@ class PADnoteParameters:public Presets
         //! RT sample data
         Sample sample[PAD_MAX_SAMPLES];
 
+        // getters/setters for "sample"
+        float* curSample(std::size_t idx);
+        const float* curSample(std::size_t idx) const;
+        const float* curSampleToPlay(std::size_t idx) const;
+        void setCurSample(std::size_t idx, float* vals);
+
+        int curSampleSize(std::size_t idx) const {
+            return sample[idx].size;
+        }
+        void setCurSampleSize(std::size_t idx, int val) {
+            sample[idx].size = val;
+        }
+        float curBaseFreq(std::size_t idx) const
```

Comment:
Seems like we might want to force this one to inline.

Reply:
We could ask the compiler to `inline`, but FWIR most compilers do their own thing anyways. I think in the end, some runtime tests should be done to see if there's any significant slowdown.